app/controllers/two_factor_authentication/webauthn_verification_controller.rb

```diff
@@ -20,6 +20,9 @@ def confirm
       handle_webauthn_result(result)
     end
 
+    def error
+    end
+
```

Comment:
Should this be removed?

```suggestion
```

Reply:
I think we like to keep the stubs of controllers?

Reply:
Ah, right. Still catches my eye as being out of place, but makes sense. 👍

Reply:
We could add `render 'error'` to make it clear?

Reply:
> We could add `render 'error'` to make it clear?

I don't think it's necessary. It's fine as-is, though if anything, I do feel like the `def error; end` syntax is at least a bit more explicit of a method being intentionally empty.